3drepo/3drepo.io
# Issue 4219 - Sort tree elements alphabetically

This fixes #4219

#### Description
Tree Sibling are sorted alphabetically (when the search mode has some filters on, the tree is flattened and all the elements are considered siblings)

#### Test cases
Open different containers and federations in the viewer and then expand the tree card.
- Check the tree is sorted;
- Expand some parent nodes and check that children are recursively sorted;
- Use the search tool and confirm that the all the displayed elements are sorted.
